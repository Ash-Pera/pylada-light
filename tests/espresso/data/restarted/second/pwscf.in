&control
    prefix = 'pwscf'
    outdir = '/private/var/folders/nm/dt02g3fd1bv0lk862vtvycnc0000gn/T/pytest-of-mdavezac/pytest-30/test_check_restarted_run0/second',
/

&system
    ecutwfc = 12.0
    ibrav = 0
    celldm = 10.2
    nat = 2
    ntyp = 1
/

&electrons
    startingwfc = 'file'
/
ATOMIC_SPECIES
Si 1.0 Si.pz-vbc.UPF

ATOMIC_POSITIONS alat
Si 3.788779800000e-02 4.389063400000e-02 -7.705962000000e-03
Si -8.916344800000e-02 -7.625391800000e-02 3.750669130000e-01

K_POINTS
2
0.25 0.25 0.75 3.0
0.25 0.25 0.25 1.0

CELL_PARAMETERS alat
-0.47850393 0.190599885 0.089913323
0.147268726 -0.420145837 0.14945944
0.420073018 0.457421551 0.592147578

ATOMIC_FORCES
Si -1.620000000000000e-05 -2.770000000000000e-05 6.010000000000000e-06
Si 1.620000000000000e-05 2.770000000000000e-05 -6.010000000000000e-06

