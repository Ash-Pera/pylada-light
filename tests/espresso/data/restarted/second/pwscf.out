
     Program PWSCF v.5.3.0 (svn rev. 11974) starts on  2May2016 at 15:54:53 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors
     Waiting for input...
     Reading input from standard input

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum          81      81     27                 1015     1015     187



     bravais-lattice index     =            0
     lattice parameter (alat)  =      10.2000  a.u.
     unit-cell volume          =     179.3769 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =         8.00
     number of Kohn-Sham states=            4
     kinetic-energy cutoff     =      12.0000  Ry
     charge density cutoff     =      48.0000  Ry
     convergence threshold     =      1.0E-06
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      =  SLA  PZ   NOGX NOGC ( 1  1  0  0 0 0)

     celldm(1)=  10.200000  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  -0.478504   0.190600   0.089913 )  
               a(2) = (   0.147269  -0.420146   0.149459 )  
               a(3) = (   0.420073   0.457422   0.592148 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = ( -1.876310 -0.144476  1.442671 )  
               b(2) = ( -0.424390 -1.899743  1.768576 )  
               b(3) = (  0.392021  0.501437  1.023316 )  


     PseudoPot. # 1 for Si read from file:
     /Users/mdavezac/workspaces/data/espresso/upf_files/Si.pz-vbc.UPF
     MD5 check sum: 6dfa03ddd5817404712e03e4d12deb78
     Pseudo is Norm-conserving, Zval =  4.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  431 points,  2 beta functions with: 
                l(1) =   0
                l(2) =   1

     atomic species   valence    mass     pseudopotential
        Si             4.00     1.00000     Si( 1.00)

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Si  tau(   1) = (   0.0378878   0.0438906  -0.0077060  )
         2           Si  tau(   2) = (  -0.0891634  -0.0762539   0.3750669  )

     number of k points=     2
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.2500000   0.2500000   0.7500000), wk =   1.5000000
        k(    2) = (   0.2500000   0.2500000   0.2500000), wk =   0.5000000

     Dense  grid:     1015 G-vectors     FFT dimensions: (  12,  12,  20)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         0.01 Mb     (     127,    4)
        NL pseudopotentials             0.02 Mb     (     127,    8)
        Each V/rho on FFT grid          0.04 Mb     (    2880)
        Each G-vector array             0.01 Mb     (    1015)
        G-vector shells                 0.00 Mb     (     508)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         0.03 Mb     (     127,   16)
        Each subspace H/S matrix        0.00 Mb     (      16,   16)
        Each <psi_i|beta_j> matrix      0.00 Mb     (       8,    4)
        Arrays for rho mixing           0.35 Mb     (    2880,    8)

     Initial potential from superposition of free atoms

     starting charge    7.99901, renormalised to    8.00000
     Starting wfc from file

     total cpu time spent up to now is        0.0 secs

     Self-consistent Calculation

     iteration #  1     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  9.0

     total cpu time spent up to now is        0.1 secs

     total energy              =     -14.31910815 Ry
     Harris-Foulkes estimate   =     -14.49124093 Ry
     estimated scf accuracy    <       0.31745269 Ry

     iteration #  2     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  3.97E-03,  avg # of iterations =  3.0

     total cpu time spent up to now is        0.1 secs

     total energy              =     -14.38718433 Ry
     Harris-Foulkes estimate   =     -14.41422503 Ry
     estimated scf accuracy    <       0.04497522 Ry

     iteration #  3     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  5.62E-04,  avg # of iterations =  3.0

     total cpu time spent up to now is        0.1 secs

     total energy              =     -14.40311453 Ry
     Harris-Foulkes estimate   =     -14.40547785 Ry
     estimated scf accuracy    <       0.00484675 Ry

     iteration #  4     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  6.06E-05,  avg # of iterations =  5.0

     total cpu time spent up to now is        0.1 secs

     total energy              =     -14.40522513 Ry
     Harris-Foulkes estimate   =     -14.40557968 Ry
     estimated scf accuracy    <       0.00222104 Ry

     iteration #  5     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  2.78E-05,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.1 secs

     total energy              =     -14.40498807 Ry
     Harris-Foulkes estimate   =     -14.40528742 Ry
     estimated scf accuracy    <       0.00099807 Ry

     iteration #  6     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.25E-05,  avg # of iterations =  5.5

     total cpu time spent up to now is        0.2 secs

     total energy              =     -14.40530298 Ry
     Harris-Foulkes estimate   =     -14.40547009 Ry
     estimated scf accuracy    <       0.00098416 Ry

     iteration #  7     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.23E-05,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -14.40515418 Ry
     Harris-Foulkes estimate   =     -14.40531970 Ry
     estimated scf accuracy    <       0.00051709 Ry

     iteration #  8     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  6.46E-06,  avg # of iterations =  4.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -14.40529267 Ry
     Harris-Foulkes estimate   =     -14.40530607 Ry
     estimated scf accuracy    <       0.00007880 Ry

     iteration #  9     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  9.85E-07,  avg # of iterations =  3.5

     total cpu time spent up to now is        0.2 secs

     total energy              =     -14.40530293 Ry
     Harris-Foulkes estimate   =     -14.40530806 Ry
     estimated scf accuracy    <       0.00003779 Ry

     iteration # 10     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  4.72E-07,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -14.40529413 Ry
     Harris-Foulkes estimate   =     -14.40530341 Ry
     estimated scf accuracy    <       0.00002287 Ry

     iteration # 11     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  2.86E-07,  avg # of iterations =  4.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -14.40530079 Ry
     Harris-Foulkes estimate   =     -14.40530064 Ry
     estimated scf accuracy    <       0.00000685 Ry

     iteration # 12     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  8.56E-08,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -14.40530039 Ry
     Harris-Foulkes estimate   =     -14.40530107 Ry
     estimated scf accuracy    <       0.00000733 Ry

     iteration # 13     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  8.56E-08,  avg # of iterations =  1.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -14.40529939 Ry
     Harris-Foulkes estimate   =     -14.40530050 Ry
     estimated scf accuracy    <       0.00000575 Ry

     iteration # 14     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  7.19E-08,  avg # of iterations =  2.0

     total cpu time spent up to now is        0.2 secs

     total energy              =     -14.40529943 Ry
     Harris-Foulkes estimate   =     -14.40529978 Ry
     estimated scf accuracy    <       0.00000189 Ry

     iteration # 15     ecut=    12.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  2.37E-08,  avg # of iterations =  1.5

     total cpu time spent up to now is        0.2 secs

     End of self-consistent calculation

          k = 0.2500 0.2500 0.7500 (   127 PWs)   bands (ev):

    -7.2765   1.5603   9.1164  14.2653

          k = 0.2500 0.2500 0.2500 (   127 PWs)   bands (ev):

    -7.1798   1.5963   9.2429  13.2048

     highest occupied level (ev):    14.2653

!    total energy              =     -14.40529911 Ry
     Harris-Foulkes estimate   =     -14.40529954 Ry
     estimated scf accuracy    <       0.00000096 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =       5.40283666 Ry
     hartree contribution      =       2.06425079 Ry
     xc contribution           =      -5.33700220 Ry
     ewald contribution        =     -16.53546277 Ry
     External forces energy    =       0.00007840 Ry

     convergence has been achieved in  15 iterations

     Writing output data file pwscf.save

     init_run     :      0.04s CPU      0.03s WALL (       1 calls)
     electrons    :      0.20s CPU      0.18s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
     potinit      :      0.01s CPU      0.01s WALL (       1 calls)

     Called by electrons:
     c_bands      :      0.15s CPU      0.14s WALL (      15 calls)
     sum_band     :      0.03s CPU      0.02s WALL (      15 calls)
     v_of_rho     :      0.01s CPU      0.01s WALL (      16 calls)
     mix_rho      :      0.01s CPU      0.01s WALL (      15 calls)

     Called by c_bands:
     init_us_2    :      0.00s CPU      0.00s WALL (      60 calls)
     cegterg      :      0.15s CPU      0.14s WALL (      30 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :      0.10s CPU      0.07s WALL (     121 calls)
     g_psi        :      0.00s CPU      0.00s WALL (      91 calls)
     cdiaghg      :      0.04s CPU      0.06s WALL (     119 calls)

     Called by h_psi:
     add_vuspsi   :      0.00s CPU      0.00s WALL (     121 calls)

     General routines
     calbec       :      0.00s CPU      0.00s WALL (     121 calls)
     fft          :      0.01s CPU      0.01s WALL (      63 calls)
     fftw         :      0.11s CPU      0.07s WALL (     788 calls)
     davcio       :      0.00s CPU      0.00s WALL (       4 calls)

     Parallel routines
     fft_scatter  :      0.00s CPU      0.00s WALL (     851 calls)

     PWSCF        :     0.29s CPU         0.25s WALL


   This run was terminated on:  15:54:53   2May2016            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
